Library/Formula/nyancat.rb

```diff
@@ -0,0 +1,18 @@
+class Nyancat < Formula
+  desc "Nyancat in your terminal, rendered through ANSI escape sequences."
+  homepage "https://github.com/klange/nyancat"
+  url "https://github.com/klange/nyancat/archive/0a7bfc16be777263841b512e1f2a9144afeb0752.tar.gz"
+  sha256 "900da8bc30d443528534c31078dc25c9b5d73c851e761e288734ca5855f40870"
+
+  head "https://github.com/klange/nyancat.git"
+
+  def install
+    system "make"
+    bin.install "src/nyancat"
+    man1.install "nyancat.1"
```

Comment:
Please use the `make install` and submit a patch to allow a custom prefix: https://github.com/klange/nyancat/blob/master/Makefile#L35-L37